code/src/components/ListOfThoughts.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { formatDistance } from 'date-fns';
+// npm install date-fns
+
+export const ListOfThoughts = ({ loading, listOfThoughts, onThoughtLikeChange }) => {
+  if (loading) {
+    return <h1 className="loading">Loading in progress...</h1>
+  }
+
+  return (
+    <section className="thought-list-container">
+      {listOfThoughts.map((thought) => (
+        <div key={thought._id} className="thought-msg">
+          <h4>{thought.message}</h4>
+          <div className="like-stats-container">
+            <button
+              type="button"
+              className={thought.hearts === 0 ? 'empty-btn' : 'like-btn'}
+              onClick={() => onThoughtLikeChange(thought._id)}>
+              <span className="heart" role="img" aria-label="heart symbol">
+              ❤️
+              </span>
+            </button>
```

Comment:
I like this piece of code. I was unsure how to make the like-button and make it change color. This together with the CSS is neat and easy to understand.